src/world/footpath.c

```diff
@@ -1472,16 +1472,25 @@ void sub_6A759F()
  */
 static void footpath_unown(int x, int y, rct_map_element *pathElement)
 {
-	int ownershipUnk = 0;
-	int z = pathElement->base_height;
 	rct_map_element *surfaceElement = map_get_surface_element_at(x >> 5, y >> 5);
```

Comment:
I do `>> 5` everywhere myself until we have a helper method. I should have never made this function accept 256 number range.